modules/ROOT/pages/salesforce-releases-migrations.adoc

```diff
@@ -0,0 +1,24 @@
+= Salesforce Releases and Migrations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+The Salesforce organization dedicated to your API community follows the standard https://trailhead.salesforce.com/en/content/learn/modules/sf_releases/sf_releases_start[Salesforce release process]. This automatically upgrades your organization to the latest Salesforce release version three times each year. As an organization administrator, you'll receive email notifications about the changes at least three months in advance, including any required steps for you to follow, guidance, and other resources.
+
+If any additional steps are required to prepare Anypoint API Community Manager, they will be provided from your MuleSoft customer success representative and on https://status.mulesoft.com/[trust.mulesoft.com]. Many Salesforce upgrades do not require any additional steps for API Community Manager.
```

Comment:
passive, present
```suggestion
Additionally, if you must take steps to prepare Anypoint API Community Manager for the upgrade, your MuleSoft customer success representative provides them and you can also view them on https://status.mulesoft.com/[trust.mulesoft.com]. Many Salesforce upgrades do not require any additional steps for API Community Manager.
```
You could probably omit the second sentence, because the first specifies "if," which implies this?